app/views/listings/_table.html.erb

```diff
@@ -4,8 +4,9 @@
       <tr>
         <th>Title</th>
         <th>Price</th>
-        <th>Bart</th>
-        <th>Bart</th>
+        <th>Transit</th>
+        <th>Walk/Transit/Bike</th>
+        <th>Crime</th>
```

Comment:
Maybe: `Crime (Personal / Property)`? I couldn't figure out what the slashes were.